cjworkbench/i18n/catalogs/util.py

```diff
@@ -104,10 +135,14 @@ def read_po_catalog(filename: str) -> Catalog:
         return read_po(catalog_file)
 
 
-def write_po_catalog(filename: str, catalog: Catalog, **kwargs):
+def write_po_catalog(filename: Union[str, pathlib.Path], catalog: Catalog, **kwargs):
```

Comment:
In our own internal APIs, permissive function signatures aren't worth the risk. Please nix the `Union` and be opinionated about type. (`pathlib.Path` is the one I'd pick, and I'd call the argument `path`.)